apps/assessment_registry/schema.py

```diff
@@ -344,23 +359,57 @@ def resolve_score_ratings(root, info, **kwargs):
     def resolve_score_analytical_density(root, info, **kwargs):
         return ScoreAnalyticalDensity.objects.filter(assessment_registry=root)
 
+    @staticmethod
+    def resolve_cna(root, info, **kwargs):
+        return Answer.objects.filter(assessment_registry=root)
+
+    @staticmethod
+    def resolve_summary_pillar_meta(root, info, **kwargs):
+        return Summary.objects.get(assessment_registry=root)
+
+    @staticmethod
+    def resolve_summary_sub_pillar_issue(root, info, **kwargs):
+        return SummarySubPillarIssue.objects.filter(assessment_registry=root)
+
+    @staticmethod
+    def resolve_summary_dimmension_meta(root, info, **kwargs):
+        return SummaryFocus.objects.filter(assessment_registry=root)
+
+    @staticmethod
+    def resolve_summary_sub_dimmension_issue(root, info, **kwargs):
+        return SummarySubDimmensionIssue.objects.filter(assessment_registry=root)
+
 
 class AssessmentRegistryListType(CustomDjangoListObjectType):
     class Meta:
         model = AssessmentRegistry
         filterset_class = AssessmentRegistryGQFilterSet
 
 
-class Query:
+class ProjectQuery:
     assessment_registry = DjangoObjectField(AssessmentRegistryType)
     assessment_registries = DjangoPaginatedListObjectField(
         AssessmentRegistryListType,
         pagination=NoOrderingPageGraphqlPagination(
             page_size_query_param='pageSize',
         )
     )
-    assessment_reg_options = graphene.Field(AssessmentRegistryOptionsType)
+    assessment_registry_options = graphene.Field(AssessmentRegistryOptionsType)
+
+    @staticmethod
+    def resolve_assessment_registries(root, info, **kwargs):
+        return get_assessment_registry_qs(info)
 
     @staticmethod
-    def resolve_assessment_reg_options(root, info, **kwargs):
+    def resolve_assessment_registry_options(root, info, **kwargs):
         return AssessmentRegistryOptionsType
+
+
+class Query():
+    issue = DjangoObjectField(IssueType)
+    issues = DjangoPaginatedListObjectField(
+        IssueListType,
+        pagination=PageGraphqlPagination(
+            page_size_query_param='pageSize'
+        )
+    )
```

Comment:
The term issue is too generic to be used in the root. Let's add a prefix. Something like this
```suggestion
    assessment_reg_summary_issue = DjangoObjectField(AssessmentRegistrySummaryissueType)
    assessment_reg_summary_issues = DjangoPaginatedListObjectField(
        AssessmentRegistrySummaryissueListType,
        pagination=PageGraphqlPagination(
            page_size_query_param='pageSize'
        )
    )
```